sweetercat/utils.py

```diff
@@ -60,6 +60,8 @@ def readSC():
                  'feh', 'feherr', 'mass', 'masserr']
         cache.set('starDB', df, timeout=5*60)
         cache.set('starCols', plots, timeout=5*60)
+    if nrows is not None:
```

Comment:
This is a nice solution, but it does not include the "speed up" that comes naturally from the nrows option in pandas.
Where it only reads in the X number of rows.
```
In [19]: %timeit a=pd.read_table("WEBSITE_online.rdb", names=names, nrows=20)
1.82 ms ± 22.2 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [20]: %timeit a=pd.read_table("WEBSITE_online.rdb", names=names)
9.78 ms ± 45.6 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```
It is only 8 ms, and I am not sure how this would change if the table is in cache. Although for testing I am not sure if it should be.